app/views/users/followers.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>followers</h1>
+<% current_user.accepted_received_follow_requests.each do |list| %>
```

Comment:
There is a more concise way to write this each loop. Review the user accessor methods in your user model.